tools/myst/host/package.c

```diff
@@ -37,15 +39,15 @@ _Static_assert(PAGE_SIZE == 4096, "");
     "\
 \n\
 Usage:\n\
-    %s package-sgx [options] <app_dir> <pem_file> <config>\n\
+    %s package-sgx [options] <app_path> <pem_file> <config>\n\
     %s package-sgx [options] <pem_file> <config>\n\
 \n\
 Where:\n\
     package-sgx -- create an executable package to run on the SGX platform\n\
                    from an application directory, package configuration and\n\
                    system files, signing and measuring all enclave resident\n\
                    pieces during in the process\n\
-    <app_dir>   -- application directory with files for root filesystem\n\
+    <app_path>  -- application directory or root filesystem (EXT2 or CPIO)\n\
```

Comment:
Probably should document what happens when <app_path> is not provided.

Reply:
I backed out support for an app_path (and restored app_dir).